jenkins/python-setup.sh

```diff
@@ -0,0 +1,29 @@
+_prefix="${1}"
+
+if [[ -z "${_prefix}" ]]; then
+    printf -- "Missing prefix argument!" >&2
+    exit 1
+fi
+
+pip3 install --no-cache-dir --progress-bar off --no-color --prefix="${_prefix}" \
+    -r ${progdir}/lint-requirements.txt \
+    -r ${progdir}/server/requirements.txt \
+    -r ${progdir}/agent/test-requirements.txt
+    # -r ${progdir}/agent/requirements.txt \
+    # -r ${progdir}/server/test-requirements.txt \
+    # -r ${progdir}/docs/requirements.txt \
```

Comment:
Do we want to retain the commented-out lines? (Default answer is "no". 😀 )

Reply:
Let's merge, we can fix in a follow-up commit.

Reply:
> Do we want to retain the commented-out lines? (Default answer is "no". 😀 )

No.  Well, yes.  But, no.  Um...maybe?  🤷

As you say, the default is "no", and I'm generally against stuff like this; however, it was just annoying enough flipping back and forth between `b0.69` and `main` that I felt like it was reasonable to have a record here of what things look(ed) like in `main`, so I left it in.  (I figured I could/would delete it if someone actually objected, but...for better or for worse, you objection was not "timely"...so it will be retained for now.)